src/plugins/saved_objects_management/server/routes/scroll_count.ts

```diff
@@ -46,24 +46,18 @@ export const registerScrollForCountRoute = (router: IRouter) => {
     },
     router.handleLegacyErrors(async (context, req, res) => {
       const { client } = context.core.savedObjects;
-      const namespaces = [];
-      if (req.body.namespacesToInclude && req.body.namespacesToInclude.length > 0) {
-        req.body.namespacesToInclude.forEach((ns) => {
-          if (ns === null) {
-            namespaces.push('default');
-          } else {
-            namespaces.push(ns);
-          }
-        });
-      }
+      const counts = {
+        type: {},
+      };
 
       const findOptions: SavedObjectsFindOptions = {
         type: req.body.typesToInclude,
         perPage: 1000,
       };
 
-      if (namespaces.length > 0) {
-        findOptions.namespaces = namespaces;
+      if ('namespacesToInclude' in req.body) {
```

Comment:
This tells me it is okay with `req.body.namespacesToInclude = null`; is that what you intended here?

Reply:
This field is only in the request if namespaces have been registered. If the request has this field and no namespaces are requested than the behavior should return a data structure like:

```
{ 
  'types': {
     'config': 0,
     ...,
  },
  'namespaces': { }
}
```

Reply:
Changed this an the other instance to `if (!!req.body.namespacesToInclude) { ...`

If `namespacesToInclude` is in the body of the request it should be a non-empty array of namespaces.

Reply:
@AMoo-Miki I had to refactor this to ensure that any filter in `visibleNamespaces` should also be present in `availableNamespaces` to ensure that a user can't type in a query string and query for a namespace that has not been registered. This is consistent with the experience for `type` as well.